src/@lekoarts/gatsby-theme-minimal-blog/components/tags.tsx

```diff
@@ -25,20 +32,27 @@ const Tags = ({ list }: MBTagsProps) => {
       </Heading>
```

Comment:
This one should return translated version of the tag as a h1
![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/34664f21-ebd2-473b-bec7-18c4cc165c73)